src/typography/titleStrong/story.tsx

```diff
@@ -1,13 +1,12 @@
 import React from 'react'
-import { boolean, text, withKnobs } from '@storybook/addon-knobs'
+import { boolean, text } from '@storybook/addon-knobs'
 import { storiesOf } from '@storybook/react'
 
 import { color } from '../../_utils/branding'
 import { BaseSection as Section } from '../../layout/section/baseSection'
 import { TextTitleStrong } from './index'
 
-const stories = storiesOf('Brand|typography', module)
-stories.addDecorator(withKnobs)
+const stories = storiesOf('Design Tokens|Typography/', module)
```

Comment:
Do we need the 'Design Tokens' level ? could we just go for Typography ?
It's not clear what's a design token (the meaning might vary depending on which medium article you read :/)

Reply:
A design token is an 'atom' basically. We can't do any design without colors, type, scalling... 
We can redefine a new taxonomy. I'm going to open a JIRA for that.